docs/signature.md

```diff
@@ -0,0 +1,93 @@
+# Image Signature Specification
+
+**Version 0.1**
+
+## Introduction
+
+This document defines a detached container image signature object and signing methods.
+
+## Signature Format
+
+```js
+{
+    "critical": {/* required fields */
```

Comment:
>  I agree it is super bizaare putting this in the schema

Do you disagree with the forward-compatibility goals (see https://github.com/containers/image/pull/59#discussion_r86182312 ), or with the implementation?